lib/rsc/common.ts

```diff
@@ -0,0 +1,76 @@
+import { storyblokInit as sbInit } from '@storyblok/js';
+import type {
+  ISbStoryData,
+  SbReactComponentsMap,
+  SbReactSDKOptions,
+  StoryblokClient,
+} from '../types';
+
+let storyblokApiInstance: StoryblokClient = null;
+const componentsMap: Map<string, React.ElementType> = new Map<string, React.ElementType>();
+let enableFallbackComponent: boolean = false;
+let customFallbackComponent: React.ElementType = null;
+
+// Cache for storyblok stories to allow for live editing
+globalThis.storyCache = new Map<string, ISbStoryData>();
```

Comment:
What is the reason we need to add it to the global namescape?

Reply:
it's because we need an in-memory storage of the story object coming from the Bridge when calling the live update server action, the server component will re-render and it will check this global cache before printing the story content